File: nomad-samples/js/account-updates/helpers/get_countries.js
export default async function getCountries(AUTH_TOKEN)
{
    // Create header for the request
    const HEADERS = new Headers();
    HEADERS.append("Content-Type", "application/json");
    HEADERS.append("Authorization", `Bearer ${AUTH_TOKEN}`);

    // Post
    const RESPONSE = await fetch(`https://dev-05.demos.media/config/ea1d7060-6291-46b8-9468-135e7b94021b/lookups.json`, {
        method: "GET",
        headers: HEADERS
    }).catch((exception) => {
        throw exception;
    });

    // Check for success
    if (RESPONSE.ok) {
        // Get the JSON from the response
        const INFO = await RESPONSE.json();

        return INFO[5];
    }

    // There was an error
    await apiExceptionHandler(RESPONSE, "Get countries failed");
}

File: nomad-samples/python/account_updates/helpers/get_countries.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import json, requests

def get_countries(AUTH_TOKEN):

    if (not AUTH_TOKEN):
        raise Exception("Authentication Token: The authentication token is invalid")
  
    API_URL = "https://dev-05.demos.media/config/ea1d7060-6291-46b8-9468-135e7b94021b/lookups.json"
  
    # Create header for the request
    HEADERS = {
        "Content-Type": "application/json"
    }

    try:
        RESPONSE = requests.get(API_URL, headers=HEADERS)
        if RESPONSE.status_code != 200:
            raise Exception("Response returned " + str(RESPONSE.status_code))

        return json.loads(RESPONSE.text)[5]
    except:
        api_exception_handler(RESPONSE, "Get countries failed")